src/site/antora/modules/ROOT/pages/manual/appenders/delegating.adoc

```diff
@@ -265,31 +265,10 @@ The `SpinPolicy` to apply, when adding elements to the queue.
 .Additional dependencies are required to use `DisruptorBlockingQueue`
 [%collapsible]
 =====
-[tabs]
-====
-Maven::
-+
-[source,xml,subs="+attributes"]
-----
-<dependency>
-  <groupId>com.conversantmedia</groupId>
-  <artifactId>disruptor</artifactId>
-  <version>{conversant-version}</version>
-  <scope>runtime</scope>
-</dependency>
-----
-
-Gradle::
-+
-[source,groovy,subs="+attributes"]
-----
-runtimeOnly 'com.conversantmedia:disruptor:{conversant-version}'
-----
-
-====
+include::partial$features/queue-conversant.adoc[]
```

Comment:
Shouldn't we be including `components/queue-conversant.adoc` instead?

Reply:
I am using the following convention:

* `components` contains only coordinates to artifacts from the Log4j distribution.
* `features` contains the coordinates of the artifact you need to add to enable certain feature.

So `features/queue-conversant.adoc` will contain the coordinates of `log4j-conversant` in 3.x and `com.conversantmedia:disruptor` in 2.x.